lib/companies/schema/industry.ex

```diff
@@ -2,13 +2,16 @@ defmodule Companies.Schema.Industry do
   use Ecto.Schema
   import Ecto.Changeset
 
-  alias Companies.Schema.Company
+  alias Companies.Schema.{Company, PendingChange}
 
   schema "industries" do
     field :name, :string
 
     has_many :companies, Company
 
+    field :is_deleted, :boolean
```

Comment:
Since this is a boolean and the `is_` prefix isn't common in Elixir, could we just call this `deleted`?

This feedback would apply to all 3 tables.

Given #493 we might want to avoid changing Industry at all since it'll likely be deleted instead? Whatcha think?

Reply:
yeah, soft-delete does not fit industry very well. However, I don't like the fact that we still have changesets on it and it can be deleted from UI, so I suggest still having these changes on industry.